python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -1052,8 +1133,8 @@ def test_get_description_batch_multiple_versions(arctic_library):
     to_append_df.index.rename("named_index", inplace=True)
     lib.append("symbol3", to_append_df)
 
-    infos_multiple_version = lib.get_description_batch([ReadRequest("symbol1", as_of=0), ReadRequest("symbol2", as_of=0), ReadRequest("symbol3", as_of=0), 
-                                                        ReadRequest("symbol1", as_of=1), ReadRequest("symbol2", as_of=1), ReadRequest("symbol3", as_of=1)])
+    infos_multiple_version = lib.get_description_batch([ReadInfoRequest("symbol1", as_of=0), ReadInfoRequest("symbol2", as_of=0), ReadInfoRequest("symbol3", as_of=0), 
+                                                        ReadInfoRequest("symbol1", as_of=1), ReadInfoRequest("symbol2", as_of=1), ReadInfoRequest("symbol3", as_of=1)])
 
     infos = infos_multiple_version[3:6]
     original_infos = infos_multiple_version[0:3]
```

Comment:
It would be good to add some tests to verify that `get_description_batch` follows the same semantics as `read_batch_metadata`. Ie, what happens if a symbol or a version doesn't exist?

Reply:
No. We are raising an exception, as I reuses the existent get_info C++ logic, which raises an exception too. But it is something we also need to address, along with the rest of methods we agreed. Good point!

Reply:
done